index.bs

```diff
@@ -156,53 +160,53 @@ spec:fetch; type:dfn; text:value
   </section>
   <section>
     <h3 id="policies">Policies</h3>
-    <p>A {{Document}} has a <dfn>feature policy</dfn>, which consists of:</p>
+    <p>A <dfn>feature policy</dfn> is a struct with the following items:</p>
     <ul>
-      <li>A set of <a data-lt="inherited policy set">inherited policies</a>.
+      <li>An <a data-lt="inherited policy">inherited policy</a>.
       </li>
       <li>A <a data-lt="declared policy">declared policy</a>.
       </li>
     </ul>
-    <p>An <dfn>empty feature policy</dfn> has an <a>inherited policy set</a>
+    <p>An <dfn>empty feature policy</dfn> has an <a>inherited policy</a>
```

Comment:
Should this be "an empty feature policy is a feature policy that has"?

Reply:
Yep. Done.